platform/libs.testng/nbproject/org-netbeans-libs-testng.sig

```diff
@@ -1262,55 +1309,29 @@ meth public abstract !hasdefault java.lang.String[] dependsOnMethods()
 meth public abstract !hasdefault java.lang.String[] groups()
 meth public abstract !hasdefault long timeOut()
 
-CLSS public abstract interface !annotation org.testng.annotations.Configuration
```

Comment:
There is unlikely much we could do about these 3rd party API removals than to accept them as fact.